### PR TITLE
Fix paths in benchmark runner

### PR DESCRIPTION
**Pull request by aliak00**
On osx I was getting a “No such file” exception for dante.txt
because CWD depended on where I executed `rdmd benchmark.d` from.
And the object files were generated in a directory also depending
on that.

This fixes obj dir to same level as bin dir within the benchmarks
dir, and allows the runCmd to take a workDir parameter for running
the benchmark executables

### CODE REVIEW

**Comment by andralex** [comment]
cc @wilzbach @MartinNowak

File: benchmark/runbench.d [hunk 1]
```diff
@@ -109,7 +111,7 @@ void runTests(Config cfg)
         foreach (_; 0 .. cfg.repeat)
         {
             sw.reset;
-            auto output = runCmd(cmd, cfg.verbose);
+            auto output = runCmd(cmd, cfg.verbose, cwd);
```
Comment (wilzbach) [inline benchmark/runbench.d]: Why `cwd`? How about `__FILE_FULL_PATH__.dirName`?

Comment (wilzbach) [reply benchmark/runbench.d]: Argh - I didn't see that `cwd` is already set to exactly this.

**Comment by wilzbach** [comment]
CircleCi failure is unrelated - see https://github.com/dlang/druntime/pull/2048 for details.

**Comment by aliak00** [comment]
Sweet! Thanks 😄

**Pull request merged**